app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    current_user = User.find(params[:user_id])
+    current_post = Post.find_by(id: params[:post_id])
+    like = Like.new(user: current_user, post: current_post)
+    current_post.save
+    like.save
+    return unless like.save
+
+    redirect_to "/users/#{current_user.id}/posts/#{params[:post_id]}"
+  end
```

Comment:
- [ ] @AlselwiAisha Great work adding methods to add a new like, I would suggest you consider adding failure and success paths, this is a good practice that enables us to handle errors while creating a new like.